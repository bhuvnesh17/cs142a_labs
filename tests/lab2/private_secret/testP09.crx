// syntax error on if-statement

func id(id1:void,id2:void,id3:void) : type {
  if a < b {}
  if (a>b) {} else {}

  if a > b
  else {}
}

