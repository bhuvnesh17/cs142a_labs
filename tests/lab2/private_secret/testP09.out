Error parsing file.
SyntaxError(8,3)[Expected OPEN_BRACE but got ELSE.]
SyntaxError(8,3)[Could not complete parsing.]
